projects/observability/src/pages/custom-dashboards/custom-dashboard-panel/custom-dashboard-panel.component.ts

```diff
@@ -94,19 +95,23 @@ export class CustomDashboardPanelComponent implements OnInit {
       value: SPAN_SCOPE
     }
   ];
+
   public constructor(private readonly exploreVisualizationBuilder: ExploreVisualizationBuilder) {}
+
   public ngOnInit(): void {
     this.panel!.series = this.tryDecodeExploreSeries(this.panel!.series);
-    if (this.panel?.interval !== 'AUTO') {
-      this.panel!.interval = this.panel!.interval;
-      this.panel!.interval = new TimeDuration(this.panel!.interval.value, this.panel!.interval.unit);
+
+    if (this.panel?.interval !== 'AUTO' && this.panel?.interval !== undefined) {
+      this.panel.interval = this.panel.interval;
```

Comment:
I know this is an existing line, but why is this needed?
```javascript
this.panel.interval = this.panel.interval
```

This seems unnecessary, can you check once?

Reply:
Good catch! The value is immediately getting assigned in the next line so this line seems redundant to me too. Removed. ✅